src/main/java/main/model/db/imports/ImportHandlers/JavaJUnitTestNG.java

```diff
@@ -15,15 +15,17 @@
 import java.util.Date;
 import java.util.List;
 
-public class JavaJUnitTestNG extends Handler{
+public class JavaJUnitTestNG extends Handler {
     private JavaJUnitTestNGHandler handler;
 
     public JavaJUnitTestNG(File file, TestNameNodeType testNameNodeType, Date finishTime) throws AqualityException {
         handler = new JavaJUnitTestNGHandler(testNameNodeType, finishTime);
         try {
             this.parser.parse(file, handler);
         } catch (SAXException | IOException e) {
-            throw new AqualityException("Cannot Parse TestNG file");
+            throw new AqualityException(String.format("Cannot parse file '%1$s'. Error: %2$s",
```

Comment:
let's add full information here ) 
like 
Cannot parse TestNG file '%1$s'. Error: %2$s"

Reply:
cause of current implementation we have single class to parse Junit and TestNg (see class is called JavaJUnitTestNG). That is why I've deleted informarion about type (Junit or TestNg) - it will be strange to see error about TestNG when I import Junit